lib/game.rb

```diff
@@ -0,0 +1,76 @@
+require_relative 'player'
+
+class Game
+  attr_reader :player_one
+  attr_reader :player_two
+  attr_reader :result
+  attr_reader :current_turn
+  attr_reader :spock_game
+
+  def initialize(player_one, player_two, spock_game)
+    @player_one = player_one
+    @player_two = player_two
+    @current_turn = @player_one
+    @spock_game = spock_game
+  end
+
+  def spock_game?
+    @spock_game
+  end
+
+  def spock_update(value)
+    @spock_game = value
+  end
+
+  def switch_turn
```

Comment:
Great that you are handling whoms turn it is in the Game class and not in the views or the controller.